app/controllers/payment-links/metadata/metadata-form.test.js

```diff
@@ -85,7 +85,12 @@ describe('Payment link metadata form model', () => {
       'test 7 header': 'test 7 value',
       'test 8 header': 'test 8 value',
       'test 9 header': 'test 9 value',
-      'test 10 header': 'test 10 value'
+      'test 10 header': 'test 10 value',
+      'test 11 header': 'test 10 value',
+      'test 12 header': 'test 10 value',
+      'test 13 header': 'test 10 value',
+      'test 14 header': 'test 10 value',
+      'test 15 header': 'test 10 value'
```

Comment:
Should the values be `test 10 value`, `test 11 value`, `test 12 value` etc?

Reply:
Values can be anything. It's the keys that need to be different. For consistency I've updated the values though.